pandas/core/arrays/numeric.py

```diff
@@ -199,3 +199,12 @@ def reconstruct(x):
             return tuple(reconstruct(x) for x in result)
         else:
             return reconstruct(result)
+
+    def __neg__(self):
```

Comment:
Thanks for moving it here! You can now also remove those in integer.py

Reply:
Removed definitions from `pandas/core/arrays/integer.py`